### PR TITLE
replace spec file with link

### PR DESCRIPTION
**Pull request by andreipope**
replace spec file with link

### CODE REVIEW

**Comment by andreipope** [comment]
@harsh-px  please, take a look at this PR.

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusAndGrafana.1.md [hunk 1]
```diff
@@ -493,6104 +493,7 @@ Navigate to the Prometheus web UI by going to `http://<master_ip>:<service_nodep
...
-                  "params": [
-                    "10s"
-                  ],
-                  "type": "time"
-                },
-                {
-                  "params": [
-                    "host"
-                  ],
-
```
Comment (harsh-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusAndGrafana.1.md]: can u create a PD ticket to change this to a single click and download?

**Comment by harsh-px** [comment]
Also cherry-pick

**Comment by andreipope** [comment]
> Also cherry-pick

@harsh-px done.

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusAndGrafana.1.md (hunk 1 above)
Comment (andreipope) [reply content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/monitoring/monitoring-px-prometheusAndGrafana.1.md]: @harsh-px  here it is https://portworx.atlassian.net/browse/PD-149

**Pull request merged**